spec/controllers/person/tags_controller_spec.rb

```diff
@@ -135,6 +136,10 @@
   end
 
   describe "DELETE #destroy" do
+    let!(:subscription_tag) { SubscriptionTag.create!(excluded: false, subscription_id: Subscription.first.id, tag_id: test_tag.id) }
+    let!(:test_tagging) { ActsAsTaggableOn::Tagging.create!(tag_id: test_tag.id, taggable_type: "Person", taggable_id: top_leader.id, context: "tags") }
+    let!(:test_tag) { ActsAsTaggableOn::Tag.create!(name: "Test") }
```

Comment:
I know that for "tag" we have fabricators, if we have fabricators for the other records too, please use fabricators over .create!. This removes a lot of code around the stuff happening before the actual spec.

Reply:
We could, for reasons of unity I will keep the creates here, since the other tags and taggables are created in the same matter in this controller test.